assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -428,13 +450,15 @@ const Trip = ({
   tripTimeBasedStyle,
   vehicleOrGhost,
   view,
+  routes,
 }: {
   trip: Trip | AsDirected
   previousEndTime: Time | undefined
   sequence: "first" | "middle" | "last"
   tripTimeBasedStyle: TimeBasedStyle
   vehicleOrGhost: VehicleOrGhost
   view: "run" | "block"
+  routes: Route[] | null
```

Comment:
There's a lot of prop drilling to do this feature. Did you consider whether a context would fit well?

Reply:
I didn't, that sounds like a good idea.

Reply:
also, filing away term "prop drilling" for future use.